### PR TITLE
Add a minimal model organization associated to user.

### PR DESCRIPTION
**Pull request by djacomy**
-Define a project model.
- Define the organization model which can be a Company or a Finess Etablissement
- Add backend function to create organization or platform with a user.

### CODE REVIEW

File: app/model/abc.py [hunk 1]
```diff
@@ -108,3 +108,10 @@ def process_bind_param(self, value, dialect):
     def process_result_value(self, value, dialect):
         return self.choices[value]
 
+
+class Base(db.Model, BaseModel):
+    __abstract__ = True
+
+    created_at = db.Column(db.DateTime, default=db.func.now())
```
Comment (antoan2) [inline app/model/abc.py]: Shouldn't we enforce `nullable=False` to the creation timestamps.

File: app/model/orga.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    __tablename__ = 'orga_address'
+
+    to_json_filter = ('id', "organization", )
+    print_filter = ('organization', )
+
+    id = db.Column(db.Integer, primary_key=True)
+    street = db.Column(db.String(50))
+    zipcode = db.Column(db.String(10))
+    city = db.Column(db.String(50))
+    lon = db.Column(db.Integer)
```
Comment (antoan2) [inline app/model/orga.py]: These should be float I think.

File: app/model/orga.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+    @classmethod
+    def get_name(cls, value):
+        return cls(value).name
+
+    def __int__(self):
+        return self.value
+
+
+class Platform(Base):
```
Comment (antoan2) [inline app/model/orga.py:23]: In the future, we might want to add contacts information for the Platform.

File: app/repository/user.py [hunk 4]
```diff
@@ -1,4 +1,5 @@
 from model.user import User
```
Comment (antoan2) [inline app/repository/user.py]: Just a question, as you are importing all the models in `model.__init__.py`, why not using just:
```suggestion
from model import User
```

**Review by antoan2 — APPROVED** [review]
I approve this first iteration.
None of my comments are blockers.

File: app/repository/user.py (hunk 4 above)
Comment (djacomy) [reply app/repository/user.py]: FIx

File: app/model/orga.py (hunk 3 above)
Comment (djacomy) [reply app/model/orga.py:23]: When we need contact, we add it.

File: app/model/orga.py (hunk 2 above)
Comment (djacomy) [reply app/model/orga.py]: FIx

File: app/model/abc.py (hunk 1 above)
Comment (djacomy) [reply app/model/abc.py]: FIx.

**Pull request merged**